.github/workflows/generate_schema.yml

```diff
@@ -41,22 +49,32 @@ jobs:
             MYSQL_ROOT_PASSWORD: root
             MYSQL_DATABASE: root
 
-    steps:    
+    steps:
     - name: Checkout Shopware
       uses: actions/checkout@v3
       with:
-          repository: shopware/platform
+          repository: shopware/shopware
           ref: trunk
           fetch-depth: 0
 
     - name: configure job
       run: |
         if [ "${{ inputs.api_type }}" = "storeapi" ]; then
-          echo "SCHEMA_FLAGS=-s openapi3 --store-api storeapi.json" >> $GITHUB_ENV
-          echo "SCHEMA_FILE=storeapi.json" >> $GITHUB_ENV
+          if [ "${{ inputs.plugin_name }}" ]; then
```

Comment:
If I understand correctly, we'll end up with 4 schemas (base + 2 for each plugin)?
 - adminapi
 - storeapi
 - DSL-adminapi
 - DSL-storeapi

Wouldn't it be easier to add `SCHEMA_FLAGS` and `SCHEMA_FILE` to the `input`?

Reply:
It would be 6 - depending on that, we do want to split up the commercial (see initial question above).